### DISCUSSION TITLE
Can you help me understand why it is necessary to use pgpool in transaction mode?

### DISCUSSION THREAD

**Discussion by davecramer** [go-pg/pg#1854] [Q&A] [answered]
see $subject

Also has https://agroal.github.io/pgagroal/ been tested

**Comment by vmihailenco** [comment, marked as answer]
PgBouncer, not pgpool. And it is not a requirement - any other mode/proxy is fine too.

I just meant to say that go-pg does not use prepared statements so it can be used with proxies like PgBouncer. I better remove that sentence.

>Also has https://agroal.github.io/pgagroal/ been tested

Not by me.

**Reply by davecramer** [reply]
Ah yes, I mistyped pgpool, meant pgbouncer. So transaction mode is not a requirement. Thanks

**Reply by kidoman** [reply]
@davecramer So what mode did you end up running PGB in? I thought it was supposed to be tx mode?

**Reply by davecramer** [reply]
I'm not running it. Mine was just a question
